### PR TITLE
PSQLADM-389 Missing files in the ProxySQL package

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

File: rpm/proxysql.spec [hunk 1]
```diff
@@ -73,14 +75,17 @@ install -m 0640 %SOURCE3 %{buildroot}/%{_sysconfdir}
 
 install -d %{buildroot}/var/lib/proxysql
 install -d %{buildroot}/var/run/proxysql
+install -d %{buildroot}/var/run/pxc_scheduler_handler
 install -m 0775 %SOURCE1 %{buildroot}/%{_bindir}/proxysql-admin
 install -m 0775 %SOURCE8 %{buildroot}/%{_bindir}/proxysql-admin-common
+install -m 0775 %SOURCE11 %{buildroot}/%{_bindir}/proxysql-common
```
Comment (mohitj1988) [inline rpm/proxysql.spec]: One question: Why the file permission is 0775, why not 0755? All standard packages have 0755.

Comment (adivinho) [reply rpm/proxysql.spec]: It looks like a misprint in initial commits.
I will fix permissions. 
Screenshot 2022-06-09 at 21 40 35

File: scripts/build_binary.sh [hunk 2]
```diff
@@ -124,18 +126,18 @@ mkdir "$INSTALLDIR"
...
             git submodule update --init
-            cd percona-scheduler
-            /usr/local/go/bin/go build -v -a -o pxc_scheduler_handler
+            sudo bash -x build_scheduler.sh
             ldd -v pxc_scheduler_handler
-            cd ../
         cd ../
         install -m 0775 proxysql-admin-tool/proxysql-admin $INSTALLDIR/usr/bin/proxysql-admin
         install -m 0775 proxysql-admin-tool/proxysql-admin-common $INSTALLDIR/usr/bin/proxysql-admin-common
+        install -m 0775 proxysql-admin-tool/proxysql-common $INSTALLDIR/usr/bin/proxysql-common
```
Comment (mohitj1988) [inline scripts/build_binary.sh]: Please change the file permissions here as well for all of them to 0755

**Review by mohitj1988 — APPROVED** [review]
Rest all looks good.

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**